extract_endpoint/requirements.txt

```diff
@@ -1,4 +1,8 @@
+astroid==1.5.3
```

Comment:
Ah yes, I knew I had encountered this error before (twice, actually), but couldn't remember how I had solved it. This is the right solution :)